src/Jobs/SweepNodeWalletsJob.cs

```diff
@@ -148,9 +148,17 @@ public async Task Execute(IJobExecutionContext context)
             _logger.LogError("{JobName} failed on node with id: {NodeId}, reason: node not found",
                 nameof(SweepNodeWalletsJob),
                 managedNodeId);
-            throw new ArgumentException("node not found", nameof(node));
+            await context.Scheduler.DeleteJob(context.JobDetail.Key, context.CancellationToken);
+            return;
```

Comment:
jobs weren't cancelled when a node was deleted so it kept giving this error forever

Reply:
Nice catch